packages/worker/package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
```

Comment:
Huh, I didn't realize this file is committed.
It can probably be deleted and added to `.gitignore`:
```
packages/*/package-lock.json
```

Reply:
Done!